### ISSUE TITLE
Use cocoa pods instead of Carthage?

### ISSUE THREAD

**Issue by Ge0rges** [udark/underdark-cocoa#8]
Is it possible to use cocoa pods instead of Carthage with under dark? Our large project uses already cocoa pods and integrating Carthage would just be another hassle.

**Comment by virl** [comment]
@Ge0rges No, because CocoaPods have flawed linking architecture, which is gradually deprecated by Carthage and SwiftPackageManager.

You don't need to "integrate Carthage", because all the "Carthage integration" there is just linking with framework.
